packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -172,6 +173,21 @@ class VMService {
       });
     }
 
+    _peer.registerMethod('getFlutterVersion', (rpc.Parameters params) async {
```

Comment:
Add a test for this. @jonahwilliams any pointers on the best way to write a test for this in flutter_tools?
I don't see any other tests in flutter_tools that directly test add service extensions to the VM.

Reply:
The vmservice as is very under-tested, I wouldn't feel too bad about skipping the unit test - provided that something somewhere is calling this method.

(As part of some Q4 refactors I'd like to replace it with the sdk packaged vmservice)

Reply:
switching to vmservice in Q4 would be great!